drafting.typ

```diff
@@ -192,45 +200,145 @@
   }
 }
 
+/// get direction of text based on defaults for the language
+/// https://github.com/typst/typst/blob/521ceae889f15f2a93683ab776cd86a423e5dbed/crates/typst-library/src/text/lang.rs#L109
+#let text-direction(dir, lang) = if dir == auto {
+  if lang in (
+    "ar", "dv", "fa", "he", "ks", "pa", "ps", "sd", "ug", "ur", "yi",
+  ) { rtl } else { ltr }
+} else {
+  dir
+}
```

Comment:
Using
```typ
#set text(dir: rtl)
```
generally breaks the layout.
This also applies to the current (`0.2.1`) version, so maybe it's for another PR...

Reply:
The function is invoked in a `context` block. I did not add `context` here because I thought it always returned `content` which I've just read is wrong. I can change this function to be `context` aware as well if you'd like.

What I meant by breaking the layout is a general issue though.

Reply:
No need to add additional context since it's a private method 👍